src/client_api/JsonRpcServer.js

```diff
@@ -168,13 +168,6 @@ class JsonRpcServer extends EventEmitter {
   /** MUST for runtime manager (main controller)*/
   setChannel(communicator) {
     this._communicator = communicator;
-    this._communicator.setOnMessage((envelop)=>{
-      const concreteCmd = envelop.content().type;
-      const action = this._actions[concreteCmd];
-      if (action) {
-        action.execute(envelop);
-      }
-    });
   }
 }
 
```

Comment:
Please explain?

Reply:
The `action` [pattern](https://github.com/enigmampc/enigma-p2p/blob/master/docs/ARCHITECTURE.md#command-pattern-and-actions) used in most of the our "runtimes", such as [node-controller](https://github.com/enigmampc/enigma-p2p/blob/master/src/worker/controller/NodeController.js#L111), [core runtime](https://github.com/enigmampc/enigma-p2p/blob/master/src/core/CoreRuntime.js#L31), [main-controller](https://github.com/enigmampc/enigma-p2p/blob/master/src/main_controller/MainController.js#L54) is simply not used in [json-rpc](https://github.com/enigmampc/enigma-p2p/blob/master/src/client_api/JsonRpcServer.js) . So if we'll need to add `actions` to it, we can bring this code back.. Currently it is mainly confusing.

Reply:
@assafmo , could you please approve?